docs/13-walkthroughs/3-add-logging.md

```diff
@@ -1,39 +1,59 @@
-# Add Logging
+# Add Logging to an Existing Zarf Cluster with Zarf Init
+
+## Introduction
 
 In this walkthrough, we are going to show how you can use a Zarf component to inject zero-config, centralized logging into your Zarf cluster.
 
 More specifically, you'll be adding a [Promtail / Loki / Grafana (PLG)](https://github.com/grafana/loki) stack to the [Doom Walkthrough](./2-deploying-doom.md) by installing Zarf's "logging" component.
 
+## System Requirements
 
-## Walkthrough Prerequisites
-1. The [Zarf](https://github.com/defenseunicorns/zarf) repository cloned: ([`git clone` Instructions](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository))
-1. Zarf binary installed on your $PATH: ([Install Instructions](../3-getting-started.md#installing-zarf))
+- You'll need an internet connection to grab the Zarf Init Package if it's not already on your machine.
 
+## Prerequisites
 
-## Install the logging component
+Prior to this walkthrough you'll want to have a working cluster with Zarf initialized
+1.  Zarf binary installed on your $PATH: ([Installing Zarf](../3-getting-started.md#installing-zarf))
+2. [Initialize a cluster](./1-initializing-a-k8s-cluster.md).
 
-To install the logging component, follow the [Initializing a Cluster Instructions](./1-initializing-a-k8s-cluster.md), but instead answer `y` when asked to install the `logging` component
+## Walkthrough
+[![Walkthrough: Add Logging to an existing Zarf Cluster with Zarf Init](../.images/walkthroughs/logging_thumbnail.png)](https://youtu.be/FYkafs1idlQ "Add Logging to an existing Zarf Cluster with Zarf Init")
 
+1. Run the `zarf init` command on your cluster.
 
-## Note the credentials
+```sh
+zarf init
+```
 
-Review the `zarf init` command output for the following:
+2. When prompted to deploy the package select `y` for Yes, then hit the `enter` 
+key. <br/> When prompted to deploy the logging component select `y` for Yes, then hit the `enter` key.
 
-![logging-creds](../.images/walkthroughs/logging_credentials.png)
+![The zarf operator is propted to deploy the zarf package and the logging component, selecting 'y' for yes, for each of them.](../.images/walkthroughs/logging_prompts.png)
```

Comment:
```suggestion
![The zarf user is prompted to deploy the zarf package and the logging component, selecting 'y' for yes, for each of them.](../.images/walkthroughs/logging_prompts.png)
```

user is probably better than operator since zarf operator could mean something else to a k8s minded person.

Reply:
Also can we make this image an iframe too?